libbpf-rs/src/object.rs

```diff
@@ -8,13 +8,13 @@ use std::path::Path;
 use std::ptr;
 
 use bitflags::bitflags;
-use nix::{errno, fcntl, unistd};
+use nix::errno;
 use num_enum::TryFromPrimitive;
 
 use crate::util;
 use crate::*;
 
-/// Sets options for opening a [`Object`]
+/// Builder for creating an [`OpenedObject`]. Typically the entry point into libbpf-rs.
```

Comment:
not a native English speaker, but I think it's not correct to use "opened" as an adjective... OpenObject is shorter as well :)